audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/AudioRecorderService.kt

```diff
@@ -1,17 +1,19 @@
-package org.odk.collect.audiorecorder.recording.internal
+package org.odk.collect.audiorecorder.recording
 
 import android.app.Service
 import android.content.Intent
 import android.os.IBinder
 import org.odk.collect.async.Cancellable
 import org.odk.collect.async.Scheduler
-import org.odk.collect.audiorecorder.getComponent
+import org.odk.collect.audiorecorder.AudioRecorderDependencyComponentProvider
 import org.odk.collect.audiorecorder.recorder.Output
 import org.odk.collect.audiorecorder.recorder.Recorder
+import org.odk.collect.audiorecorder.recording.internal.RecordingForegroundServiceNotification
+import org.odk.collect.audiorecorder.recording.internal.RecordingRepository
 import java.io.Serializable
 import javax.inject.Inject
 
-internal class AudioRecorderService : Service() {
+class AudioRecorderService : Service() {
```

Comment:
Annoyingly things that we inject into can't be `internal` any longer because they appear in the component's `inject` methods. This is a weird side effect but not really a problem.